third_party/BUILD

```diff
@@ -469,11 +469,12 @@ UNNECESSARY_DYNAMIC_LIBRARIES = select({
     "//src/conditions:arm": "*.so *.jnilib *.dll",
     "//src/conditions:linux_aarch64": "*.so *.jnilib *.dll",
     "//src/conditions:linux_ppc": "*.so *.jnilib *.dll",
-    "//src/conditions:freebsd": "*.so *.jnilib *.dll",
-    "//src/conditions:openbsd": "*.so *.jnilib *.dll",
```

Comment:
Why are you removing these?

Reply:
Um... because despite all my efforts not to I managed to overlook something when I ran the push?  I’ll un-remove them now...

Reply:
No worries, thanks!

Reply:
@philwo Patched up.